scripts/zones/Lufaise_Meadows/mobs/Flockbock.lua

```diff
@@ -1,11 +1,45 @@
 -----------------------------------
 -- Area: Lufaise Meadows
 --   NM: Flockbock
+-- https://www.bg-wiki.com/ffxi/Flockbock
+-- TODO: NM has several possible spawn locations
+-----------------------------------
+mixins = { require('scripts/mixins/rage') }
 -----------------------------------
 local entity = {}
 
+entity.onMobInitialize = function(mob)
+    mob:addMod(xi.mod.BINDRES,      100)
```

Comment:
are you trying to make immune, or just highly resistant?

Reply:
Those set to 100 will make themeffectively immune to those status effects.
However, those mods are used for the resist job traits, and shouldnt be used for this.

I kind of rememver we got functionality to set immunities via script. Key word: __kind__

Reply:
theres a field that takes them as a bitmask in mob_pools table, and the script ability was added so that we coudl both add and remove them because some monsters _change_ their immunities mid fight. I wouldn't mind converting it all to script and eliminating the filed but that'd be a lot of work and might cause some non-nm to have scripts that otherwise wouldn't need them, so for now I still recommend using pools to static immunities.

If he just meant to add really potent resistance tho, then its a matter of is-this-a-trait and is-this-the-right-amount

Reply:
I've removed these modifiers in favor of adding the immunities directly to mob_pools.